src/App/component/Login.jsx

```diff
@@ -32,7 +34,14 @@ export default function Login() {
   const history = useHistory()
   const [username, setUsername] = useState("")
   const [password, setPassword] = useState("")
-  const [invalidAccount, setInvalidAccount] = useState(false)
+  const [accountOperationHint, setAccountOperationHint] = useState("")
+  const [open, setOpen] = useState(false);
```

Comment:
Suggest clarifying who use these states.
```suggestion
  const [isWhatOpen, setWhatOpen] = useState(false);
```

Reply:
不是改成 What 的吧？ XD

Reply:
Of course my friend, it is just an example.

Reply:
OMG, he did it ....
@jn2657 Please don't use `What` for this naming, it is just an example, the real name should not be `What` ...